### ISSUE TITLE
RFC Compiler options should be grouped under a `thrift` key

### ISSUE THREAD

**Issue by dantswain** [pinterest/elixir-thrift#194]
Right now `thrift_files` and `thrift_output` have to go right in the project config's root namespace.  I think it would be cleaner if they were nested under a `thrift` key:

```
def project do
  # ..
  thrift: [
    thrift_files: Path.wildcard("thrift/*.thrift"),
    thrift_output: "gen-lib"
  ]
end
```

Doing this it might make sense to change the names of the parameters to not have a redundant "thrift_" prefix.

By way of prior art: This is the way dialyxir works, and it's fairly intuitive and unobtrusive.

**Comment by jparise** [comment]
That's an interesting idea. I think there's prevent both ways (with the `erl*` and `elixir*` keys), but this could be a nice improvement.

I can look into this in a couple of weeks. It would be nice to hear any other thoughts in the interim.

**Comment by dantswain** [comment]
It looks like the changes would be pretty easy to make and I'd be glad to do it if we get agreement.

ExDoc also uses this approach (`docs: []`).  Hex sort of uses this approach with the `package: []` key.

**Comment by scohen** [comment]
I'm a fan, as long as we don't stutter `thrift: [files: ...]` as opposed to  `thrift: [thrift_files:...]`

**Comment by jparise** [comment]
Implemented by #197
